plugins/woocommerce/client/blocks/assets/js/blocks/product-reviews/inner-blocks/review-rating/edit.tsx

```diff
@@ -0,0 +1,94 @@
+/**
+ * External dependencies
+ */
+import clsx from 'clsx';
+import { __, sprintf } from '@wordpress/i18n';
+import { useEntityProp } from '@wordpress/core-data';
+import {
+	AlignmentToolbar,
+	BlockControls,
+	useBlockProps,
+} from '@wordpress/block-editor';
+import { useStyleProps } from '@woocommerce/base-hooks';
+import type { BlockEditProps } from '@wordpress/blocks';
+
+/**
+ * Renders the `woocommerce/product-review-rating` block on the editor.
+ *
+ * @param   {Object}  props                    React props.
+ * @param   {Object}  props.attributes         Block attributes.
+ * @param   {Object}  props.setAttributes      Function to set block attributes.
+ * @param   {Object}  props.context           Inherited context.
+ * @param   {string}  props.context.commentId The comment ID.
+ *
+ * @return {JSX.Element} React element.
+ */
+export default function Edit( {
+	context: { commentId },
+	attributes,
+	setAttributes,
+}: BlockEditProps< {
+	textAlign: string;
+} > & {
+	context: { commentId: string };
+} ) {
+	const { textAlign } = attributes;
+	const styleProps = useStyleProps( attributes );
+	const className = clsx(
+		styleProps.className,
+		'wc-block-components-product-review-rating',
```

Comment:
Nit: Is there any reason to have components in this className? 

```suggestion
		'wc-block-product-review-rating',
```

Reply:
Nope, removed it: https://github.com/woocommerce/woocommerce/pull/56332/commits/ab16e279c648dd788b7cafea8172c02e9538e9fa